mkosi.md

```diff
@@ -797,183 +804,258 @@ details see the table below.
   output formats `gpt_ext4`, `gpt_xfs`, `gpt_btrfs`. Defaults to 3G.
 
-`--esp-size=`
+`ESPSize=`, `--esp-size=`
+
+: Similar to `RootSize=`, configures the size of the UEFI System
+  Partition (ESP). This is only relevant if the `Bootable=` option is
+  used to generate a bootable image. Defaults to 256 MB.
 
-: Similar, and configures the size of the UEFI System Partition
-  (ESP). This is only relevant if the `--bootable` option is used to
-  generate a bootable image. Defaults to 256M.
+`SwapSize=`, `--swap-size=`
 
-`--swap-size=`
+: Similar to `RootSize=`, configures the size of a swap partition on
+  the image. If omitted, no swap partition is created.
 
-: Similar, and configures the size of a swap partition on the
-  image. If omitted no swap partition is created.
+`HomeSize=`, `--home-size=`
 
-`--home-size=`
+: Similar to `RootSize=`, configures the size of the `/home`
+  partition. If omitted, no separate `/home` partition is created.
 
-: Similar, and configures the size of the `/home` partition. If
-  omitted no separate `/home` partition is created.
+`SrvSize=`, `--srv-size=`
 
-`--srv-size=`
+: Similar to `RootSize=`, configures the size of the `/srv`
+  partition. If omitted, no separate `/srv` partition is created.
 
-: Similar, and configures the size of the `/srv` partition. If
-  omitted no separate `/srv` partition is created.
+### [Validation] Section
 
-`--checksum`
+`Checksum=`, `--checksum`
 
 : Generate a `SHA256SUMS` file of all generated artifacts after the
   build is complete.
 
-`--sign`
+`Sign=`, `--sign`
 
 : Sign the generated `SHA256SUMS` using `gpg` after completion.
 
-`--key=`
+`Key=`, `--key=`
 
 : Select the `gpg` key to use for signing `SHA256SUMS`. This key
   is required to exist in the `gpg` keyring already.
 
-`--bmap`
+`BMap=`, `--bmap`
 
 : Generate a `bmap` file for usage with `bmaptool` from the generated
   image file.
 
-`--password=`
+`Password=`, `--password=`
 
 : Set the password of the `root` user. By default the `root` account
   is locked. If this option is not used but a file `mkosi.rootpw` exists
   in the local directory the root password is automatically read from it.
 
-`--password-is-hashed`
+`PasswordIsHashed=`, `--password-is-hashed`
 
 : Indicate that the password supplied for the `root` user has already been
-  hashed, so that the string supplied with `--password` or `mkosi.rootpw` will
+  hashed, so that the string supplied with `Password=` or `mkosi.rootpw` will
   be written to `/etc/shadow` literally.
 
-`--autologin`
+`Autologin=`, `--autologin`
 
-: Enable autologin for the `root` user on pts/0 (nspawn), tty1 (QEMU) and
-  ttyS0 (QEMU with `--qemu-headless`) by patching `/etc/pam.d/login`.
+: Enable autologin for the `root` user on `/dev/pts/0` (nspawn),
+  `/dev/tty1` (QEMU) and `/dev/ttyS0` (QEMU with `QemuHeadless=yes`)
+  by patching `/etc/pam.d/login`.
 
-`--extra-search-paths=`
+<!-- FIXME: why are all those settings in [Validation] ?! -->
```

Comment:
The groupings don't really make a lot of sense for a lot of settings. AFAIK we don't even enforce them. I wonder if we should drop them alltogether.

Reply:
I think they are useful to make it easier for humans to read the config files… I'd keep them, at least for now.